styles.css

```diff
@@ -3,37 +3,100 @@
 }
 
 body {
-    margin: 0;
     font-family: 'Roboto', sans-serif;
     background-image: url("Images/dragons.png");
     background-attachment: fixed;
 }
 
-header {
+.header {
     text-align: center;
     font-family: 'Raleway', sans-serif;
     font-weight: 400;
     text-decoration: none;
-    color: #708090;
+    color: #696969;
     font-size: 24px;
 }
 
 .dragonContainer {
+    margin: 0 auto;
 }
 
-.dragonBox {
+.outerContainer {
+    float: left;
     width: 25%;
+    padding: 15px;
+}
+
+.dragonBox {
+    padding-bottom: 15px;
+    border: 1px solid gray;
+    border-radius: 25px;
 }
 
 .dragonImage {
     height: 300px;
-    width: 300px;
+    width: 100%;
     border-radius: 25px;
 }
 
-dragonName {
+.dragonName {
     font-family: 'Raleway', sans-serif;
```

Comment:
Could just add this font family to the body instead of having to repeat it throughout code